src/jit/compiler.hpp

```diff
@@ -1052,9 +1051,6 @@ inline GenTreePtr Compiler::gtNewIconHandleNode(
 
 #if defined(LATE_DISASM)
     node = new (this, LargeOpOpcode()) GenTreeIntCon(TYP_I_IMPL, value, fields DEBUGARG(/*largeNode*/ true));
-
-    node->gtIntCon.gtIconHdl.gtIconHdl1 = handle1;
```

Comment:
Do we still need this ifdef at all? Can we create a non-large node in all cases?

Reply:
I plan on investigating this further and fixing it in an upcoming change